tests/plugins/slow_init.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+from lightning import Plugin
+import time
+
+plugin = Plugin()
+
+
+@plugin.init()
+def init(options, configuration, plugin):
+    plugin.log("slow_init.py initializing")
+    time.sleep(1)
```

Comment:
Would the test be reliable with just a 1-second sleep here?

Reply:
Should be ok, maybe `valgrind` might be a bit too slow on CI, but 1 second is usually plenty in these cases.